src/test/java/mpack/Assignement2.java

```diff
@@ -0,0 +1,59 @@
+package mpack;
+
+import java.time.Duration;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+//import org.openqa.selenium.WindowType;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+public class Assignement2 {
+	public static void main(String[] args)
+	{
+		WebDriverManager.chromedriver().setup();
+		WebDriver driver = new ChromeDriver();
+		driver.get("http://www.fb.com/");
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+		String currentUrl = driver.getCurrentUrl();
+		if(currentUrl.equals("https://www.facebook.com/"))
+		{
+			System.out.println("Redirected to correct URL:" +currentUrl);
+		}
+		else
+		{
+			System.out.println("Redirected to wrong URL:" +currentUrl);
+		}
```

Comment:
Please correct it

Variable Name
Convention: Use camelCase.

Rules:

Similar to method names, start with a lowercase letter, and capitalize subsequent words.
Variable names should be descriptive and represent the data they hold.
Boolean variables often start with "is" or "has."
Examples:

employeeName
totalAmount
isVerified

Reply:
Corrected the naming convention of methods